TechnicalDocuments/NewHiresCheckList.md

```diff
@@ -41,8 +41,6 @@ Here's how to get the iOS project up and running.
 1. Globally configure Git to use SSH instead of HTTPS:
      ```
      git config --global url."git@github.com:".insteadOf "https://github.com/"
-     git config --global url."git@github.com:".pushinsteadof "git://github.com/"
-     git config --global url."git@github.com:".pushinsteadof "https://github.com/"
```

Comment:
Those 2 are not needed. I only had to setup the first line and everything works just fine.